src/main/java/net/fabricmc/loader/impl/launch/knot/KnotClassLoader.java

```diff
@@ -81,6 +81,13 @@ public URL findResource(String name) {
 		return urlLoader.findResource(name);
 	}
 
+	@Override
+	public Enumeration<URL> findResources(String name) throws IOException {
+		Objects.requireNonNull(name);
+
+		return urlLoader.findResources(name);
```

Comment:
Should this not also include any results from `originalLoader`? Or at least if `urlLoader` doesn't return any results, like `getResources` does below.

Reply:
This current implementation is consistent with findResource(String).

Reply:
When multiple Models contain resources with the same path, findResource can only display one of them.

Reply:
> This current implementation is consistent with findResource(String).

When multiple Models contain resources with the same path, findResource can only display one of them.

Reply:
> Should this not also include any results from `originalLoader`? Or at least if `urlLoader` doesn't return any results, like `getResources` does below.

My understanding is that findResource and findResources are designed to find resources within the current ClassLoader implementation, while getResource is designed to query the parent ClassLoader and query within the current ClassLoader.

You can see that the findResources implemented by URLClassLoader does not query the results of the parent ClassLoader.